nats.go

```diff
@@ -1489,7 +1498,7 @@ func (nc *Conn) waitForExits() {
 	nc.wg.Wait()
 }
 
-// Report the connected server's Url
+// ConnectedUrl report the connected server's Url
```

Comment:
reports and Url -> URL

Reply:
Can't change this, its an existing function, I just fixed the comment to match reality

Reply:
I meant at end of the comment.

Reply:
doh, ok, fixed.